opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -47,6 +47,11 @@ namespace Opm
     public:
         typedef BaseType :: WellMapType WellMapType;
 
+        // TODO: same definition with WellInterface, eventually they should go to a common header file.
+        static const int Water = BlackoilPhases::Aqua;
```

Comment:
To be safe, just include WellInterface.hpp and use it directly (at least I think that is acceptable).

Reply:
But `WellStateFullyImplicitBlackoil` was included in `WellInterface.hpp`....

Reply:
Ok, did not notice that.